src/Compilers/Core/Portable/NativePdbWriter/PdbWriter.cs

```diff
@@ -148,6 +148,16 @@ public void SerializeDebugInfo(IMethodBody methodBody, StandaloneSignatureHandle
 
             if (blob.Length > 0)
             {
+                const int limit = 0x10_000;
+                if (blob.Length > limit)
+                {
+                    throw new SymUnmanagedWriterException(string.Format(
+                        CodeAnalysisResources.SymWriterMetadataOverLimit,
+                        methodBody.MethodDefinition,
+                        blob.Length,
+                        limit));
+                }
```

Comment:
Note: the name of the attribute is part of the limit length. Is that included in the `blob`?

Reply:
No, the name is not included in the `blob`. You are right the limit is wrong now, will fix.